jwst/pipeline/assign_wcs.cfg

```diff
@@ -1,4 +1,10 @@
 name = "assign_wcs"
 class = "jwst.assign_wcs.AssignWcsStep"
+sip_approx = True  # enables SIP approximation for imaging modes.
+sip_max_pix_error = 0.25  # imaging modes only
+sip_degree = None  # imaging modes only
```

Comment:
`None` is not an allowed value in `configobj` for a value that needs an integer.  I would recommend removing this and `sip_inv_degree` below.  And in fact, remove all the new ones here, as they are taken care of in the step's `"spec"` attribute.

This is the disadvantage of using `configobj` to do our validation and is a known issue.

Btw, this is what is causing the suffix unit test failure.

Reply:
Ok, I removed all of these parameters from the config file.